presto-main/src/test/java/io/prestosql/execution/scheduler/TestSourcePartitionedScheduler.java

```diff
@@ -396,8 +404,8 @@ public void testBlockCausesFullSchedule()
         scheduleResult = secondScheduler.schedule();
         assertFalse(scheduleResult.isFinished());
         assertTrue(scheduleResult.getBlocked().isDone());
-        assertEquals(scheduleResult.getNewTasks().size(), 3);
-        assertEquals(secondStage.getAllTasks().size(), 3);
+        assertEquals(scheduleResult.getNewTasks().size(), 0);
```

Comment:
the test does not `FullSchedule` anymore. I think this test can be removed as we now cover this test by `testNewTaskScheduledWhenChildStageBufferIsUnderutilized` and `testNoNewTaskScheduledWhenChildStageBufferIsOverutilized`.
The only difference is that this this test was inducing blocked on placements via first query

Reply:
Yes, that makes sense. `testNoNewTaskScheduledWhenChildStageBufferIsOverutilized` is using a new node to assert the full schedule.